modules/serverless-logic-install-kn-workflow-artifact-images.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-artifact-images_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin using the artifacts image
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have installed Podman on your local machine. 
+
+.Procedure
+
+. Download the Knative Workflow plugin using the `logic-kn-workflow-cli-artifacts-rhel8` image by running the following commands:
+
+.. To login to the Red{nbsp}Hat Registry run the following command on a system with podman installed:
++
+[source,shell]
+----
+$ podman login registry.redhat.io
+----
++
+You can use your Red{nbsp}Hat customer portal or a registry service account username and password.
+
+.. Start the `logic-kn-workflow-cli-artifacts-rhel8` image.
++
+[source,shell]
+----
+$ export KN_IMAGE=registry.redhat.io/openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0
+----
++
+[source,shell]
+----
+$ export KN_CONTAINER_ID=$(podman run -di $KN_IMAGE)
+----
+
+.. Copy the Knative Workflow plugin binary file according to your environment.
++
+.Binary file for Linux amd64 architecture
+[source,shell]
+----
+$ podman cp $KN_CONTAINER_ID:/usr/share/kn/linux_amd64/kn-workflow-linux-amd64.tar.gz kn-workflow-linux-amd64.tar.gz
+----
++
+.Binary file for Linux arm64 architecture
+[source,shell]
+----
+$ podman cp $KN_CONTAINER_ID:/usr/share/kn/linux_arm64/kn-workflow-linux-arm64.tar.gz kn-workflow-linux-arm64.tar.gz
+----
++
+.Binary file for macOS amd64 architecture
+[source,shell]
+----
+$ podman cp $KN_CONTAINER_ID:/usr/share/kn/macos_amd64/kn-workflow-macos-amd64.tar.gz kn-workflow-macos-amd64.tar.gz
+----
++
+.Binary file for macOS arm64 architecture
+[source,shell]
+----
+$ podman cp $KN_CONTAINER_ID:/usr/share/kn/macos_arm64/kn-workflow-macos-arm64.tar.gz kn-workflow-macos-arm64.tar.gz
+----
++
+.Binary file for Windows amd64 architecture
+[source,shell]
+----
+$ podman cp $KN_CONTAINER_ID:/usr/share/kn/windows/kn-workflow-windows-amd64.zip kn-workflow-windows-amd64.zip
+----
```

Comment:
In this step, change `[source,shell` to `[source,terminal]`. Optionally, you might consider a more typical format, as follows.

Usually, when a command has multiple options like this, we include the command a single time and explain the options as required. In this case, the source file and the target filename are the same, so you can also replace the repeated filename at the end with a `.`

In this case, something like:
```suggestion
.. Copy the Knative Workflow plugin binary file by running the following command:
+
[source,terminal]
----
$ podman cp $KN_CONTAINER_ID:/usr/share/kn/<path_to_binary> .
----
+
where `<path_to_binary>` is the path to the file for your environment:
+
[cols=2,options=header]
|===
|Environment
|Path to binary file

|Linux amd64 architecture
|`linux_amd64/kn-workflow-linux-amd64.tar.gz`

|Linux arm64 architecture
|`linux_arm64/kn-workflow-linux-arm64.tar.gz`

|macOS amd64 architecture
|`macos_amd64/kn-workflow-macos-amd64.tar.gz`

|macOS arm64 architecture
|`macos_arm64/kn-workflow-macos-arm64.tar.gz`

|Windows amd64 architecture
|`windows/kn-workflow-windows-amd64.zip`

|===
```

If you're concerned about using a partial path, you could do this:

```suggestion
.. Copy the Knative Workflow plugin binary file by running the following command:
+
[source,terminal]
----
$ podman cp $KN_CONTAINER_ID:<path_to_binary> .
----
+
where `<path_to_binary>` is the path to the file for your environment:
+
[cols=2,options=header]
|===
|Environment
|Path to binary file

|Linux amd64 architecture
|`/usr/share/kn/linux_amd64/kn-workflow-linux-amd64.tar.gz`

|Linux arm64 architecture
|`/usr/share/kn/linux_arm64/kn-workflow-linux-arm64.tar.gz`

|macOS amd64 architecture
|`/usr/share/kn/macos_amd64/kn-workflow-macos-amd64.tar.gz`

|macOS arm64 architecture
|`/usr/share/kn/macos_arm64/kn-workflow-macos-arm64.tar.gz`

|Windows amd64 architecture
|`/usr/share/kn/windows/kn-workflow-windows-amd64.zip`

|===
```